compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/LottieAnimation.kt

```diff
@@ -58,8 +58,9 @@ private fun InfiniteAnimation(animation: Animation, modifier: Modifier) {
     )
     val invalidationController = remember { InvalidationController() }
 
-    animation.seekFrameTime(time, invalidationController)
     Canvas(modifier) {
+        animation.seekFrameTime(time, invalidationController)
```

Comment:
It's not a fix, but reverting an old workaround

See the comment removed in https://github.com/JetBrains/compose-multiplatform-core/pull/1002

Reply:
without strong skipping mode unstable captures/params trigger recomposition of canvas (it has unstable animation)

but with with SSM enabled, recomposition [could be skipped](https://developer.android.com/develop/ui/compose/performance/stability/strongskipping#when-skip) for lambdas with unstable parameters, just by comparing references. hence animation is the same object (and only internal state is changed by `seekFrameTime`). to prevent that we need to add some other capture to that lambda that would either notify runtime about changes, or change its identity